### PR TITLE
docs: DEV-4013: Documentation update for ML 

### PR DESCRIPTION
**Pull request by KonstantinKorotaev**

### CODE REVIEW

File: docs/source/guide/active_learning.md [hunk 1]
```diff
@@ -76,7 +77,7 @@ In order to maximize the training efficiency and effectiveness of your machine l
 On the project data manager, select **Label All Tasks** to start labeling.
 
 As your model retrains and a new version is updated in Label Studio, the tasks shown next to annotators are always those with the lowest prediction scores, reflecting those with the lowest model certainty. The predictions for the tasks correspond to the latest model version.
-
+</div>
```
Comment (anithamuthu2022) [inline docs/source/guide/active_learning.md:80]: Read and insert <div class="enterprise-only">or <div class="opensource-only"> .
Important note: before and after div should be newlines, like on the screenshot!

Reference: https://heartex.slack.com/archives/C01HGFXA7KR/p1670507767449669

File: docs/source/guide/active_learning.md [hunk 2]
```diff
@@ -86,11 +87,13 @@ If you want to change the behavior of the active learning loop, you can make man
...
 1. Manually [retrieve predictions from a model](ml.html#Get-predictions-from-a-model).
 2. [Sort the tasks in the data manager by prediction score](manage_data.html#Example-Sort-by-prediction-score).
 3. Select **Label Tasks As Displayed** when labeling tasks. 
 
-This manual active learning loop does not automatically update the order of tasks presented to annotators as the ML backend trains with each new annotation and produces new predictions. Therefore, instead of on-the-fly automated active learning, you can perform a form of batched active learning, where you perform annotation for a period, stop to train the model, then retrieve new predictions and start annotating tasks again.
\ No newline at end of file
+This manual active learning loop does not automatically update the order of tasks presented to annotators as the ML backend trains with each new annotation and produces new predictions. Therefore, instead of on-the-fly automated active learning, you can perform a form of batched active learning, where you perform annotation for a period, stop to train the model, then retrieve new predictions and start annotating tasks again.
+</div>
```
Comment (anithamuthu2022) [inline docs/source/guide/active_learning.md:99]: Read and insert <div class="enterprise-only">or <div class="opensource-only"> .
Important note: before and after div should be newlines, like on the screenshot!

Reference: https://heartex.slack.com/archives/C01HGFXA7KR/p1670507767449669

File: docs/source/guide/custom_metric.md [hunk 3]
```diff
@@ -282,3 +283,4 @@ To grant permissions to a specific user, role, or EKS node group used to manage
    ]
 }
 ```
+</div>
```
Comment (anithamuthu2022) [inline docs/source/guide/custom_metric.md:286]: Read and insert <div class="enterprise-only">or <div class="opensource-only"> .
Important note: before and after div should be newlines, like on the screenshot!

Reference: https://heartex.slack.com/archives/C01HGFXA7KR/p1670507767449669

**Comment by anithamuthu2022** [comment]
@KonstantinKorotaev I've reviewed and shared feedback for your attention.

File: docs/source/guide/active_learning.md (hunk 1 above)
Comment (KonstantinKorotaev) [reply docs/source/guide/active_learning.md:82]: Fixed

File: docs/source/guide/custom_metric.md [hunk 4]
```diff
@@ -11,6 +11,8 @@ section: "Quality control"
 
 ---
 
+<div class="enterprise-only">
```
Comment (makseq) [inline docs/source/guide/custom_metric.md:14]: @KonstantinKorotaev  This whole page has already moved to enterprise, please remove this div out from here.

Comment (makseq) [reply docs/source/guide/custom_metric.md:14]: image

**Review by makseq — CHANGES_REQUESTED** [review]

File: docs/source/guide/custom_metric.md (hunk 4 above)
Comment (KonstantinKorotaev) [reply docs/source/guide/custom_metric.md]: Fixed

**Pull request merged**